src/explore-education-statistics-common/src/components/form/__tests__/FormComboBox.test.tsx

```diff
@@ -173,632 +179,552 @@ describe('FormComboBox', () => {
       expect(option2).toHaveAttribute('aria-selected', 'false');
       expect(option3).toHaveAttribute('aria-selected', 'true');
     });
-  });
-
-  describe('list box', () => {
-    test('pressing ArrowUp will cycle the selected item through the entire list', () => {
-      const { container, getByText, getByLabelText } = render(
-        <FormComboBox
-          id="test-combobox"
-          inputLabel="Choose option"
-          onInputChange={() => {}}
-          onSelect={() => {}}
-          options={['Option 1', 'Option 2', 'Option 3']}
-        />,
-      );
-
-      const input = getByLabelText('Choose option') as HTMLInputElement;
-
-      fireEvent.change(input, {
-        target: {
-          value: 'Test',
-        },
+    describe('list box', () => {
+      test('pressing ArrowDown will cycle the selected item through the entire list', () => {
```

Comment:
Shouldn't this test case be 'pressing ArrowUp will cycle the selected item through the entire list'?

We've already got another test case further down that is testing 'pressing ArrowDown will cycle the selected item through the entire list'

Reply:
Yep think you're right. Must have duplicated this one accidentaly